src/app/cat-filter/cat-filter.component.ts

```diff
@@ -0,0 +1,85 @@
+import { Component, OnInit, ViewChild, AfterViewInit } from '@angular/core';
+import { Store } from '@ngrx/store';
+import * as actions from '../cat-store/cat.action';
+import { selectCategoryContent, selectCategoryItems, selectLength } from '../cat-store/cat.selector';
+import { MatPaginator } from '@angular/material/paginator';
+import { FormControl, FormGroup } from '@angular/forms';
+
+@Component({
+  selector: 'app-cat-filter',
+  templateUrl: './cat-filter.component.html',
+  styleUrls: ['./cat-filter.component.scss'],
+})
+export class CatFilterComponent implements OnInit, AfterViewInit {
+  form = new FormGroup({
+    category: new FormControl(''),
+    categoryItem: new FormControl(''),
+  });
+
+  categories = [
```

Comment:
we have entity Category which is one of your categories, let's another name for this func